README.md

```diff
@@ -1,40 +1,53 @@
-Transportr
-==========
+[![Transportr Logo](/app/src/main/res/mipmap-xhdpi/ic_launcher.png)](https://transportr.app)
+
+## Transportr
 
 The public transport companion that respects your privacy and your freedom.
 Transportr is a non-profit app developed by people around the world to make using public transport as easy as possible wherever you are. 
 
-[![Transportr Logo](/app/src/main/res/mipmap-xhdpi/ic_launcher.png)](https://transportr.app)
 [![Build Status](https://travis-ci.org/grote/Transportr.svg?branch=master)](https://travis-ci.org/grote/Transportr)
+[![GitHub commits since latest release](https://img.shields.io/github/commits-since/grote/Transportr/latest)](https://github.com/grote/Transportr/releases/tag/latest) 
+[![Twitter Follow](https://img.shields.io/twitter/follow/TransportrApp?style=social)](https://twitter.com/intent/follow?screen_name=TransportrApp)
+
+## Install
+
+[![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/grote/Transportr?label=latest%20version&sort=semver)](https://github.com/grote/Transportr/releases)
+[![F-Droid](https://img.shields.io/f-droid/v/de.grobox.liberario)](https://f-droid.org/packages/de.grobox.liberario/)
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80">](https://f-droid.org/packages/de.grobox.liberario/)
+[<img src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png" alt="Google Play Download" height="80">](https://play.google.com/store/apps/details?id=de.grobox.liberario)
 
-Please **[visit the website](https://transportr.app)** for more information!
+* Pre-releases and beta versions for advanced users are available via [a special F-Droid repository](http://grobox.de/fdroid/).
 
-If you find any issues with this app, please report them at [the issue tracker](https://github.com/grote/Transportr/issues). Contributions are both encouraged and appreciated. If you like to contribute please [check the website](https://transportr.app/contribute) for more information.
+## Screenshots
 
-The upstream repository is at: https://github.com/grote/Transportr
+[<img src="https://user-images.githubusercontent.com/50620416/140822064-86778b46-f701-40b0-a9bd-fc10d1397f0f.gif" height="444">](https://transportr.app/img/transportr.webm)
```

Comment:
Where does this come from? Is it a gif or a webm video? In either case, it should be added to the repo if we want to show it in the README.